R/Utilities.R

```diff
@@ -16,7 +16,8 @@ utils::globalVariables(c("AboveWQXUpperThreshold", "ActivityIdentifier",  "Activ
                          "ResultIdentifier", "TADA.ResultMeasureValue", "n_sites",
                          "n_records", "statecodes_df", "STUSAB" ,"ActivityStartTime.Time", "numorgs", "dup_id",
                          "LatitudeMeasure", "TADA.ResultMeasureValue.DataTypeFlag", "Name", "TADA.Detection_Type",
-                         "DetectionQuantitationLimitTypeName", "TADA.Limit_Type", "multiplier"))
+                         "DetectionQuantitationLimitTypeName", "TADA.Limit_Type", "multiplier", "last_col", "summ", 
```

Comment:
I didn't add last_col and summ because they are functions, but maybe this works anyway.

Reply:
Ah okay, let's pull those back out of here.